### DISCUSSION TITLE
Find node by name

### DISCUSSION THREAD

**Discussion by paulorrfilho** [google/filament#3581] [Q&A]
Is there a way to iterate through all the meshes loaded at once (in one .obj file) to find a mesh/node by name?

**Comment by romainguy** [comment]
Filament does not load OBJ files for you. You can use a library or convert your OBJ to a simple format called "filamesh" we provide as an example (see our samples and the filamesh command line tool). If you load a glTF file using `gltfio` we do provide a way to find nodes by names.

**Reply by paulorrfilho** [reply]
Yes, I have been using filamesh to convert my obj files. My question is can I iterate through the nodes after I load the converted filamesh file in my app? Or is it considered just one single mesh?

**Reply by romainguy** [reply]
Sub-meshes are called "parts" in `filamesh`. Our sample `MeshLoader` code does not preserve the list of parts, it just directly creates a `Renderable`. `filamesh` also doesn't attempt to store mesh names, just material names. I would encourage you to copy `ModelLoader.kt` and to modify it to suit your needs.
